src/traits.rs

```diff
@@ -158,6 +158,19 @@ pub trait Integer:
     /// The value `1`.
     fn one() -> Self;
 
+    /// The value `0` with the same precision as `other`.
+    fn zero_like(other: &Self) -> Self {
+        Self::from_limb_like(Limb::ZERO, other)
+    }
+
+    /// The value `1` with the same precision as `other`.
+    fn one_like(other: &Self) -> Self {
+        Self::from_limb_like(Limb::ONE, other)
+    }
```

Comment:
I've wondered if we should have a `One` trait, similar to `num_traits::One`.

The only reason we don't use `num_traits::One` is so `is_one` can return `Choice`, as it were.

Reply:
Seems logical to have such a trait. If we're redefining `num_traits::Zero`, might as well redefine `One` too.